scripts/release_info.py

```diff
@@ -292,24 +303,111 @@ def _check_storage_models(current_release):
     return [item for item in diff_list if item.startswith('core/storage')]
 
 
+def check_blocking_bugs(repo):
+    """Checks the number of unresolved blocking bugs.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Returns:
+        int: Number of unresolved blocking bugs.
+    """
+    blocking_bugs_milestone = repo.get_milestone(number=39)
+    return blocking_bugs_milestone.open_issues
+
+
+def check_prs_for_current_release(repo):
```

Comment:
Maybe `check_prs_for_current_release_are_released` or `check_blocking_bug_fix_released`? (The current name doesn't explains what the method checks.)

Reply:
Done.